video/image_writer.c

```diff
@@ -165,7 +165,7 @@ static void prepare_avframe(AVFrame *pic, AVCodecContext *avctx,
 
 static bool write_lavc(struct image_writer_ctx *ctx, mp_image_t *image, const char *filename)
 {
-    FILE *fp = fopen(filename, "wb");
+    FILE *fp = mp_fopen(filename, "wb");
```

Comment:
Does this actually make a difference? This file already includes `osdep/io.h`, which should already map `fopen` to `mp_open`.

Reply:
I wasn't aware that redefining was happening. I'll undo that then

Reply:
> I'll undo that then

I didn't ask to undo that. I asked if it actually makes a difference, and I mentioned why I _think_ it shouldn't make a difference.

So does it or does it not make a difference?

Reply:
As long as `mp_fopen` is called on Windows it doesn't make a difference. Still undoing that though since `mp_fopen` seems to be `__MINGW32__` only, breaking the unix OS builds.